core/services/ocr2/plugins/ocr2keeper/evm21/block_subscriber_test.go

```diff
@@ -0,0 +1,56 @@
+package evm
```

Comment:
There don't seem to be any tests that ensure all subscribers get all messages. This is a critical piece of functionality and should be in a unit test.

Reply:
done